test/extensions/stats_sinks/wasm/test_data/BUILD

```diff
@@ -0,0 +1,41 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_library",
+    "envoy_package",
+)
+load("//bazel/wasm:wasm.bzl", "wasm_cc_binary")
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+filegroup(
```

Comment:
This is obsolete and only used in common/wasm because of the obsolete commit of the rust raw .wasm file.  You should be able to remove it and just depened on test_cpp.wasm.

Reply:
sounds good, fixed